nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common-base/nxserver/config/bulk-kafka-config.xml.nxftl

```diff
@@ -0,0 +1,25 @@
+<#escape x as x?xml>
+<?xml version="1.0"?>
+<component name="org.nuxeo.kafka.defaultConfig">
+<#if "${kafka.enabled}" == "true">
+  <require>org.nuxeo.runtime.stream.kafka.service</require>
+
+  <extension point="kafkaConfig" target="org.nuxeo.runtime.stream.kafka.service">
+    <kafkaConfig name="bulk" topicPrefix="${kafka.topicPrefix}bulk-">
+      <producer>
+        <property name="bootstrap.servers">${kafka.bootstrap.servers}</property>
+        <property name="default.replication.factor">${kafka.default.replication.factor}</property>
+      </producer>
+      <consumer>
+        <property name="bootstrap.servers">${kafka.bootstrap.servers}</property>
+        <property name="request.timeout.ms">${kafka.request.timeout.ms}</property>
+        <property name="max.poll.interval.ms">${kafka.max.poll.interval.ms}</property>
+        <property name="session.timeout.ms">${kafka.session.timeout.ms}</property>
+        <property name="heartbeat.interval.ms">${kafka.heartbeat.interval.ms}</property>
+        <property name="max.poll.records">${kafka.max.poll.records}</property>
+      </consumer>
+    </kafkaConfig>
```

Comment:
Could we add this config to the already existing XP in `kafka-config.xml` ? Besides you named the component with the same name as the default config one